src/main/java/com/textadventure/Main.java

```diff
@@ -5,7 +5,7 @@
 public class Main {
     public static void main(String[] args) {
         System.out.println("Hi there! Let's rock this");
-        String input = " w walk s North";
+        String input = "W walk W S S South ";
```

Comment:
If we are writing tests, then there is no need to add stuff in the main method anymore, for now this is fine, but future reference, if its not needed lets not add it.